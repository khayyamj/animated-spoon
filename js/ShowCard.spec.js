import React from 'react'
import ShowCard from './ShowCard'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

test('ShowCard snapshot test', () => {
  const component = shallow(<ShowCard />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
