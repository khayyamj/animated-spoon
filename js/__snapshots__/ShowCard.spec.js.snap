exports[`test ShowCard snapshot test 1`] = `
<div
  className="show-card">
  <img
    src="/public/img/posters/undefined" />
  <div>
    <h3 />
    <h4>
      (
      )
    </h4>
    <p />
  </div>
</div>
`;
